modules/models/src/main/java/org/eclipse/xpanse/modules/models/response/Response.java

```diff
@@ -33,6 +35,9 @@ public class Response {
     @Schema(description = "Describes if the request is successful")
     private Boolean success;
 
+    @Schema(description = "Describes if the request is successful")
+    private String trackingId;
```

Comment:
I think it is better to keep this as response header "X-TRACKING-ID' then we can send it even in positive responses as well. That's how a lot of other API systems have implemented it. 

If this a bigger change, we can keep this as it is in this PR and change it to a HTTP response header in another user story.

Reply:
The tracking id moved into headers.
Error Response:
![idea64_x2wNMwZD7K](https://github.com/user-attachments/assets/a40c1aa7-20b0-4094-a997-565a825bb49d)
Successful Response:
![idea64_0QozROUWCs](https://github.com/user-attachments/assets/1cee78b4-f141-45e8-9abe-375c31de1a1a)